### ISSUE TITLE
height property does not support % values

### ISSUE THREAD

**Issue by AlexandreSajus** [Avaiga/taipy#419]
**TLDR: The user should be able to enter CSS values for height and width in properties of all visual elements as `height=XX%`**

The height property of the chart visual element does not support values in percentage.

For example `height=10%` does not do anything:

```python
from taipy.gui import Gui
import pandas as pd

data = pd.DataFrame({"Time": [1, 2, 3], "Max AQI": [4, 5, 6]})

page = """
<|{data}|chart|height=10%|>
"""

Gui(page).run()
```

![image](https://github.com/Avaiga/taipy/assets/62465003/bcf51c72-adad-4eb2-bfae-66cf191bd120)

Currently, the workaround is `height=10vh`, but it is unintuitive.

The doc even refers to `height="10%"` [here](https://docs.taipy.io/en/latest/manuals/gui/viselements/chart/)
![image](https://github.com/Avaiga/taipy/assets/62465003/22fc2671-2edc-45e1-a6d6-6e88e8a6c5ca)
But it does not work.

**Comment by FredLL-Avaiga** [comment]
Plotly calculates the chart size at runtime depending on the content and the available space.
it cannot get a correct size if specified in % (especially vertically)
I guess we need to be more precise in the doc

**Comment by jrobinAV** [comment]
Wont't fix
